.bazelrc

```diff
@@ -19,6 +19,7 @@ build:asan --define tcmalloc=disabled
 build:asan --build_tag_filters=-no_asan
 build:asan --test_tag_filters=-no_asan
 build:asan --define signal_trace=disabled
+build:asan --test_env=ASAN_OPTIONS=detect_odr_violation=0
```

Comment:
use `detect_odr_violation=1` I don't think asan will fail with 1, but will fail with 2 (which is default)

Reply:
asan is still reporting errors with `detect_odr_violation=0`. I'll try to set it to 1 and see if it passes.